src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -115,3 +117,13 @@ export function DropdownMenuPopover(props: DropdownMenuPopoverProps) {
 export function useDropdownMenuContext() {
   return useMenuButtonContext()
 }
+
+export function DropdownDivider(props) {
+  return <div {...props} css={dropdownDividerCss} />
```

Comment:
We should use `<hr>` element as well as [`role="separator"`](https://www.digitala11y.com/separator-role/)
```suggestion
  return <hr role="separator" {...props} css={dropdownDividerCss} />
```
Then we can provide a type for its props using a generic type from React:
```typescript
export type DropdownDividerProps  = React.ComponentPropsWithoutRef<"hr">
```
This will let TS know that `DropdownDivider` accepts all the props/attributes supported by `<hr>`.

Reply:
😬 That's what I had here, first. Removed it because I wondered if this _fully_ qualifies as a "thematic break" — from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr:

> The HTML <hr> element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section.

I didn't consult the actual spec for `hr` yet.

Reply:
Then again, same source, emphasis mine:

> Historically, this has been presented as a horizontal rule or line. While it may still be displayed as a horizontal rule in visual browsers, this element is now defined in semantic terms, rather than presentational terms, **so if you wish to draw a horizontal line, you should do so using appropriate CSS.**

Reply:
I asked about this in our frontend chapter meeting as well and the general consensus was that since the `<hr />` should semantically separate content and this list would have separated content, that using the `<hr />` would be a good way to go.

Reply:
^ fixing that now! 🙏